src/data.yaml

```diff
@@ -2481,6 +2481,19 @@ jobs:
         long: 9.8011421
     review:
     rating:
+  - name: Acker e.V.
+      website: https://www.acker.co/
+      jobs: https://www.acker.co/Jobs
+      description: Innovative und praxisorientierte Konzepte für mehr Wertschätzung für Natur und Lebensmittel
+      remote: true
+      speculative: true
+      field: Misc.
+      geo:
+        - country: Germany
+          lat: 52.466230
+          long: 13.366840
+      review: https://www.kununu.com/de/acker-e-v
+      rating: '4.2/5'
```

Comment:
This needs to be changed to `4.2`.

Reply:
Thank you for your fast reply, we changed the file accordingly